project-generator/src/test/resources/io/syndesis/project/converter/test-pull-push-pom.xml

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
If you remove this line and remove it from the mustache template, the test should still work, right ? (both lines are added by this PR) because the generator adds the version literally (somehow, don't see it in the template though).

I ask because if you look into the generate pom.xml for an integration, the version is still inserted literally for the connectors and you have a property with no use.